### PR TITLE
USPs & Responsiveness

### PR DESCRIPTION
**Pull request by Resaki1**
Bildschirmfoto 2023-08-24 um 18 25 31
Bildschirmfoto 2023-08-24 um 18 25 09
Bildschirmfoto 2023-08-24 um 18 24 10
Bildschirmfoto 2023-08-24 um 18 23 46

### CODE REVIEW

**Comment by brandstetterm** [comment]
Something seems wrong with the images :( 
image

File: src/components/Uniqueness.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  useEffect(() => {
+    const theme = document.documentElement.getAttribute("theme");
+    if (theme) {
+      setTheme(theme);
+    }
+  }, []);
+
+  return (
+    <section className="uniqueness__wrapper">
+      <h2>Was macht Scrumlr so besonders?</h2>
```
Comment (brandstetterm) [inline src/components/Uniqueness.tsx:16]: This should be font-size 40px on larger displays and 24px on smaller displays

File: src/components/Uniqueness.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+      <p>
+        Scrumlr fördert eine ausgewogene Zusammenarbeit innerhalb jedes Teams.{" "}
+      </p>
```
Comment (brandstetterm) [inline src/components/Uniqueness.tsx:19]: This should be font-size 20px

File: src/components/Uniqueness.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+      <div className="uniqueness__section">
+        <div className="uniqueness__image">
+          <img
+            src={`assets/uniqueness/uniqueness_${theme}_0.webp`}
+            className="uniqueness__image-0"
+            alt="Ein Beispiel für die Auswahl unterschiedlicher Templates"
+          />
+        </div>
+        <div className="uniqueness__text">
+          <h3>Schnellstart: ohne Registrierung und kostenlos</h3>
```
Comment (brandstetterm) [inline src/components/Uniqueness.tsx:29]: This should be font-size 40px on larger displays and 16px on mobile displays

File: src/components/Uniqueness.tsx [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+          <p>
+            Starte direkt und unverbindlich, ohne aufwändige Registrierung oder
+            Anmeldung. Scrumlr ist außerdem kostenlos – für immer!
```
Comment (brandstetterm) [inline src/components/Uniqueness.tsx:32]: This should be font-size 20px on larger displays and 14px on mobile displays

File: src/components/Mobile.tsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+          className="mobile__background"
+          src={`assets/mobile/mobile_background_${theme}.svg`}
+          alt=""
+        />
+      </div>
+      <div className="mobile__text">
+        <h2>
+          Verwende Scrumlr überall und zu jeder Zeit- auch auf deinem
+          Smartphone!
+        </h2>
```
Comment (brandstetterm) [inline src/components/Mobile.tsx:32]: This should be font-size 40px on larger displays

File: src/components/Mobile.tsx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+        <p>
+          Verpasse unterwegs keine Retrospektive mehr! Verwende dazu einfach
+          Scrumlr auf deinem Smartphone.
+        </p>
```
Comment (brandstetterm) [inline src/components/Mobile.tsx:36]: This should be font-size 20px on larger displays

File: src/pages/index.astro [hunk 7]
```diff
@@ -1,18 +1,24 @@
...
+		<Choices client:visible />
+		<Uniqueness client:idle />
+		<Mobile client:idle />
   </main>
 </Layout>
 
 <style>
 	main {
-    width: 100vw;
+		padding: 1.5rem;
```
Comment (brandstetterm) [inline src/pages/index.astro:20]: This messes with the first section where the background needs to stretch to the top. I'd prefer if you remove this

**Comment by Resaki1** [comment]
> Something seems wrong with the images :( image

Thanks for noticing and pointing it out. It's fixed now:
Bildschirmfoto 2023-08-29 um 15 10 41

File: src/layouts/Layout.astro [hunk 8]
```diff
@@ -91,6 +91,33 @@ const { title } = Astro.props;
...
+		font-size: 40px;
+		line-height: 48px;
+	}
+
+	p {
+		font-size: 20px;
+		line-height: 28px;
+	}
+
+	@media screen and (max-width: 850px) {
```
Comment (brandstetterm) [inline src/layouts/Layout.astro:104]: I think we should opt for a mobile first appraoch

Comment (brandstetterm) [reply src/layouts/Layout.astro:104]: I've also used 768px, 992px and 1280px as breakpoints

**Review by brandstetterm — APPROVED** [review]

**Pull request merged**